Hotfix chart display spacing

NguyenVu11796 opened the pull request:
hotfix: chart spacing display
- add horizontalEdge variable as the space between final datapoint and the bound of the chart.
- add condition to increase spacing while spacing is < horizontalEdge

NguyenVu11796 committed:
hotfix: NOT resolved. Testing out different spacing algorithms.

NguyenVu11796 committed:
hotfix-2: add var horizontalEdge and while spacing < horizontalEdge, increase spacing

NguyenVu11796 committed:
adjusted algorithm for horizontal Edge and condition for while loop

rider3458 commented on src/vn/edu/rmit/kuri/output/Display.java:
```diff
@@ -174,7 +174,14 @@ public static void chart(Summary summary, Metric metric) {
 
-    spacing = (int) Math.floor((float) (horizontal - 1) / valueForDisplay.size());  // TODO: This is not accurate. Fix it.
+    //create spacing between data points on the chart
+    spacing = (int) Math.floor((float) (horizontal) / valueForDisplay.size());
```
I think you should keep horizontal -1 because the chart already uses 1 column for the y axis.

NguyenVu11796 replied:
True.

rider3458 commented on src/vn/edu/rmit/kuri/output/Display.java:
```diff
@@ -174,7 +174,14 @@ public static void chart(Summary summary, Metric metric) {
-    spacing = (int) Math.floor((float) (horizontal - 1) / valueForDisplay.size());  // TODO: This is not accurate. Fix it.
+    //create spacing between data points on the chart
+    spacing = (int) Math.floor((float) (horizontal) / valueForDisplay.size());
+    /*check if the remaining space between the last data point and the horizontal edge is smaller
```
I think horizontalEdge is kind of misleading, u can use something like spaceRemaining.

NguyenVu11796 replied:
remainingSpace adheres more to the naming conventions. I'll use it. Thanks!

NguyenVu11796 committed:
rename horizontalEdge to remainingSpace
adjust algorithm to spacing.

NguyenVu11796 committed:
FINAL hotfix: previous hotfix contains runtime error out of bound for last datapoint.
*This hotfix changes how spacing is calculated (exempting the vertical axis and first datapoint)
*change the condition for loop: while the remaining data points are not bigger than remaining space => increase space.

Commit:
display data points based on the min and max values instead only max
